docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,109 @@ You can follow these steps to validate all cluster update approaches.
+
+   The first profile version (1.0.0) should be the Hello Universe 1.2.0 pack with the API disabled.
+
+   The second version (1.1.0) should be the Hello Universe 1.2.0 pack with the API enabled.
+
+2. Build a cluster using the profile version 1.0.0 (takes about 15-20 min)
+3. Download the Admin kubeconfig file from the cluster Overview.
+
+   Sample output
+
+![How to download Admin kubeconfig file from Palette](/clusters_management_updates_kubeconfig-download.webp)
+
+4. Go your terminal application and run the following command
+
+`kubectl get pods --kubeconfig=admin.cluster-update-deletion.kubeconfig --namespace=hello-universe -o=custom-columns="POD_NAME:.metadata.name,CONTAINER_NAME:.status.containerStatuses[].name,CONTAINER_ID:.status.containerStatuses[].containerID"`
+
+5. This command will display the containers in the cluster that are in the hello universe namespace. It will include the
+   pod name, container name, and container ID. You can use these identifiers with various `kubectl` options.
+
+Sample output
+
+![Using kubectl to see what containers are running](/clusters_management_updates_kubectl-noapi-hellouniverse.webp)
+
+6. Return to Palette and update the cluster using the profile version 1.1.0 (takes about 1-2 min).
+7. Return to the terminal and run the following command
+
+`kubectl get pods --kubeconfig=admin.cluster-update-deletion.kubeconfig --namespace=hello-universe -o=custom-columns="POD_NAME:.metadata.name,CONTAINER_NAME:.status.containerStatuses[].name,CONTAINER_ID:.status.containerStatuses[].containerID"`
+
+Sample output
+
+![Using kubectl to see what containers are running in the updated cluster](/clusters_management_updates_kubectl-api-hellouniverse.webp)
+
+With the cluster updated, we can see that the previous version of the Hello Universe app remains. This container,
+however, is not used with the API version and is no longer needed.
+
+8. Locate the pod name of the container you want to remove and run the following command.
+
+`kubectl scale deployment hello-universe-deployment --replicas=0 hello-universe-deployment-6854947c67-jtlp7 --kubeconfig=cluster-update-deletion.kubeconfig --namespace hello-universe`
+
+Sample output
+
+![Using kubectl to remove resources no longer in use](/clusters_management_updates_kubectl-api-hellouniverse-removed.webp)
+
+This command will reduce the container count to 0, effectively stopping the container. If you have multiple clusters
+with resources no longer in use, you could put the command into a script to help remove those additional resources.
+
+### Validation
+
+You can follow these steps to validate all cluster update approaches.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+2. Navigate to the left **Main Menu** and select **Clusters**.
+3. Select the cluster you modified, and navigate to the **Workloads** tab of the cluster.
+4. Select **Pods** and filter for the deployment namespace.
+5. Confirm that the unnecessary resources have been removed.
```

Comment:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com/).

2. Navigate to the left main menu and select **Clusters**.

3. Select the cluster you modified, and navigate to the **Workloads** tab of the cluster.

4. Select **Pods** and filter for the deployment namespace.

5. Confirm that the unnecessary resources have been removed.
```

Reply:
Added some spacing and switched to a recent style guide change where "main menu" is no longer a proper noun in bold.